app/scripts/migrations/153.ts

```diff
@@ -48,6 +46,12 @@ function transformState(
     );
     return state;
   }
+
+  if (!hasProperty(state, 'TokenListController')) {
+    console.warn(`newState.TokenListController is not present`);
+    return state;
+  }
```

Comment:
Added a check for a missing property and did `console.warn` instead of `captureException`; 

I kept the captureException in the check `if (!isObject(tokenListControllerState))` since if it is not an object it means its corrupted state